README.md

```diff
@@ -76,10 +76,25 @@ config.vm.synced_folder ".", "/home/core/share", id: "core", :nfs => true,  :mou
 
 After a 'vagrant reload' you will be prompted for your local machine password.
 
-#### Provisioning with user-data
+#### Provisioning with Ignition (VirtualBox Provider (default))
 
-The Vagrantfile will provision your CoreOS VM(s) with [coreos-cloudinit][coreos-cloudinit] if a `user-data` file is found in the project directory.
-coreos-cloudinit simplifies the provisioning process through the use of a script or cloud-config document.
+When using the VirtualBox provider for vagrant (the default), Ignition is used to provision the machine. This uses a special plugin that is
+automatically installed when using the default Vagrantfile. The config options for the plugin are all prefixed with `config.ignition` and can
+be found in this Vagrantfile or in the README of the [plugin][https://github.com/coreos/vagrant-ignition].
+
+To get started, copy `config.ign.sample` to `config.ign` and make any necessary modifications.
```

Comment:
We'd have to add support for Vagrant once coreos-metadata learns how to access the metadata attributes.